docs/application-development-guide.md

```diff
@@ -3388,6 +3822,17 @@ Content is not available
 
 Content is not available
 
+</TabItem>
+<TabItem value="python">
+
+To define a Query, set the Query decorator `@workflow.query` on the Query function inside your Workflow.
+
+```python
+@workflow.signal
```

Comment:
Wrong decorator, wrong example, and need to make sure decorator indention is same as method and that it's clear this is a method. Also need to be clear that the method return is the query return.